firebase-appdistribution/test-app/test-app.gradle

```diff
@@ -60,6 +60,7 @@ dependencies {
     println("Building with HEAD version ':firebase-appdistribution' of SDK")
 
     // All variants use the API
+    implementation project(':firebase-common')
```

Comment:
We're already adding `:firebase-common:ktx` below. Do we need this too? If so please put it down next to the other common one.

Reply:
I don't need it. This was probably an artifact of a bad merge.